### PR TITLE
fix: issue#54

### PR DESCRIPTION
**Pull request by yoution**

### CODE REVIEW

**Comment by yoution** [comment]
#54

File: src/components/Pagination/styles.module.scss [hunk 1]
```diff
@@ -15,6 +16,9 @@
     border-color: #2A2A2A;
   }
 
+  > button {
+    margin-bottom: 10px !important;
```
Comment (maxceem) [inline src/components/Pagination/styles.module.scss:20]: @yoution could we please avoid using `!important` unless there is no other way as it's kind of anti-pattern and has drawbacks https://uxengineer.com/css-specificity-avoid-important-css/.

**Review by maxceem — CHANGES_REQUESTED** [review]
@yoution works great, just one code improvement below:

**Comment by maxceem** [comment]
@yoution one more thing. We use pagination not only on the teams list, but also on other pages. After this fix, the vertical alignment is broken on other pages (there is extra space under the page number now):

image


See http://localhost:8080/taas/myteams/16882

File: src/components/Pagination/styles.module.scss (hunk 1 above)
Comment (yoution) [reply src/components/Pagination/styles.module.scss]: I knew,but I  could not found a better method, but now I find a new method.

File: src/components/Pagination/styles.module.scss [hunk 2]
```diff
@@ -15,6 +17,9 @@
     border-color: #2A2A2A;
   }
 
+  > button[class] {
```
Comment (maxceem) [inline src/components/Pagination/styles.module.scss:20]: @yoution this technically works, but if someone new look at this it's not clear what does it mean and why we did so. Could we replace it with some more clear way, for example,`.page, .next, .prev { margin-bottom: 10px; }`, it's more wordy, but it's more clear.

**Review by maxceem — CHANGES_REQUESTED** [review]
@yoution there are still 2 things:
- comment below and
- https://github.com/topcoder-platform/taas-app/pull/59#issuecomment-762789512

**Comment by maxceem** [comment]
Sorry @yoution this is fixed https://github.com/topcoder-platform/taas-app/pull/59#issuecomment-762789512

So only this one is left https://github.com/topcoder-platform/taas-app/pull/59#discussion_r560161599

File: src/components/Pagination/styles.module.scss (hunk 2 above)
Comment (yoution) [reply src/components/Pagination/styles.module.scss:20]: @maxceem  ok , I will add new classname for the button
but how about #59(comment), it still not work?

Comment (maxceem) [reply src/components/Pagination/styles.module.scss:20]: @yoution this https://github.com/topcoder-platform/taas-app/pull/59#issuecomment-762789512 already works, thanks!

**Comment by maxceem** [comment]
Example link to comment https://github.com/topcoder-platform/taas-app/pull/59#issuecomment-762789512

**Review by maxceem — CHANGES_REQUESTED** [review]
@yoution after the last fix when I run `npm run dev` I get error:

image

**Comment by yoution** [comment]
@maxceem  sorry, push again

**Review by maxceem — APPROVED** [review]
Thanks for all the small fixes @yoution. Works perfect now.

**Pull request merged**